apps/meteor/server/services/messages/service.ts

```diff
@@ -134,6 +138,7 @@ export class MessageService extends ServiceClassInternal implements IMessageServ
 
 		if (!this.isEditedOrOld(message)) {
 			await Promise.all([
+				this.checkMAC.isWithinLimits({ message, room: _room }),
```

Comment:
since we're doing the "is edited" check here, I removed from the "callback code".

I think this block of code should be moved up, before the other actions.. it may have been put here due to requirement of a specific order.. need to take a look

Reply:
thx man